packages/text-field/theme/lumo/vaadin-input-field-shared-styles.js

```diff
@@ -9,187 +9,191 @@ import '@vaadin/vaadin-lumo-styles/sizing.js';
 import '@vaadin/vaadin-lumo-styles/spacing.js';
 import '@vaadin/vaadin-lumo-styles/style.js';
 import '@vaadin/vaadin-lumo-styles/typography.js';
-import '@vaadin/vaadin-lumo-styles/mixins/required-field.js';
+import { requiredField } from '@vaadin/vaadin-lumo-styles/mixins/required-field.js';
 import '@vaadin/vaadin-lumo-styles/font-icons.js';
-import '@vaadin/vaadin-lumo-styles/mixins/field-button.js';
-
-registerStyles(
-  '',
-  css`
-    :host {
-      --lumo-text-field-size: var(--lumo-size-m);
-      color: var(--lumo-body-text-color);
-      font-size: var(--lumo-font-size-m);
-      font-family: var(--lumo-font-family);
-      -webkit-font-smoothing: antialiased;
-      -moz-osx-font-smoothing: grayscale;
-      -webkit-tap-highlight-color: transparent;
-      padding: var(--lumo-space-xs) 0;
-    }
-
-    :host::before {
-      height: var(--lumo-text-field-size);
-      box-sizing: border-box;
-      display: inline-flex;
-      align-items: center;
-    }
-
-    :host([focused]:not([readonly])) [part='label'] {
-      color: var(--lumo-primary-text-color);
-    }
-
-    /* TODO: remove when the following components are updated to use new indicator:
-      combo-box, date-picker, time-picker, date-time-picker, select. */
-    [part='label']::after {
-      display: none;
-    }
-
-    :host([has-helper]) [part='helper-text']::before {
-      content: '';
-      display: block;
-      height: 0.4em;
-    }
-
-    [part='helper-text'] {
-      display: block;
+import { fieldButton } from '@vaadin/vaadin-lumo-styles/mixins/field-button.js';
```

Comment:
nit: let's place mixin imports together and move `font-icons` import above.

Reply:
Done.